assets/css/main.css

```diff
@@ -1,11 +1,8851 @@
-﻿/*!
+@charset "UTF-8";
```

Comment:
This file is supposed to be minified CSS from running the `drone exec` command. Now, this is clearly not minified.